testsuite/podman_runner/19_get_server_logs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -xe
+
+if [ $# -ne 1 ];
+then
+	echo "Usage: ${0} server_id"
+	echo "server_id is used for creating a unique folder"
+	exit -1
+fi
+
+src_dir=$(cd $(dirname "$0")/../.. && pwd -P)
+server_id=${1}
+rm -rfv /tmp/test-all-in-one/server-logs/${server_id}
+mkdir -p /tmp/test-all-in-one/server-logs/${server_id}
+sudo -i podman exec uyuni-server-all-in-one-test bash -c "tar jcvf /tmp/server-logs/${server_id}/var.log.rhn.tar.bz2 /var/log/rhn/"
+sudo -i podman exec uyuni-server-all-in-one-test bash -c "supportconfig -R /tmp/server-logs/${server_id} && chmod 644 /tmp/server-logs/${server_id}/scc_uyuni-server-all-in-one-test*.txz*"
```

Comment:
Be aware that for the supportconfig to collect proper logs from SUMA, we need to have installed the plugins for SUMA

Reply:
I checked the logs and I saw a spacewalk-debug folder. I assume it is installed, given we use the pattern to install the already released packages.